### DISCUSSION TITLE
le lien que j'ai utilisée pour le tester

### DISCUSSION THREAD

**Discussion by miquanta** [sajjadjaved01/iptv#35] [Q&A]
[](url)
![image](https://user-images.githubusercontent.com/72265650/111344831-06ea8980-867d-11eb-882c-75aa08031bd1.png)

**Comment by sajjadjaved01** [comment]
http://iptvpro1.premium-tv.media:8789/.get.php?username=amine&password=Amine1&type=m3u_plus
this link is not working.

Please post link for testing.
